common/network-common/src/main/java/org/apache/spark/network/crypto/AuthRpcHandler.java

```diff
@@ -139,8 +139,4 @@ protected boolean doAuthChallenge(
     return true;
   }
 
-  @Override
-  public MergedBlockMetaReqHandler getMergedBlockMetaReqHandler() {
-    return saslHandler.getMergedBlockMetaReqHandler();
-  }
```

Comment:
This seems to be added at Apache Spark 3.2.0 via SPARK-35671, right?

Reply:
Yes, we didn't test with encryption enabled and this was missed.